ansible/condor.yml

```diff
@@ -110,10 +124,19 @@
         state: directory
         owner: condor
         recurse: yes
+  tags:
+    - worker
 
+- name: restart condor
+  become: true
+  hosts: condor
+  gather_facts: false
+  tasks:
     - ansible.builtin.systemd:
         name: condor
         state: restarted
         enabled: yes
   tags:
     - worker
+    - manager
+    - submission
```

Comment:
I wanted to be sure that Condor was always restarted so that it would pick up any configuration changes.